### PR TITLE
Add note to README regarding boolean dependencies

### LINKED ISSUE #74: Requires or conditions not working
**What version of `electron-installer-redhat` are you using?**
    "electron-installer-redhat": "^0.5.0"
**What version of `node` and `npm` are you using?**
5.3.0
v8.1.4

**What operating system are you using?**
Linux Ubuntu 16.04 lts

**Can you compile the [example app](https://github.com/unindented/electron-installer-redhat/tree/master/example) successfully?**
yes

**What did you do? Please include the configuration you are using for `electron-installer-redhat`.**
```
{
  "productName": "Brave",
  "genericName": "Web Browser",
  "homepage": "https://brave.com/",
  "icon": "res/app.png",
  "section": "web",
  "categories": [
    "Network",
    "WebBrowser"
  ],
  "mimeType": [
    "text/html",
    "text/xml",
    "application/xhtml_xml",
    "image/webp",
    "x-scheme-handler/http",
    "x-scheme-handler/https",
    "x-scheme-handler/ftp"
  ],
  "requires": [
    "GConf2 | gconf2",
    "libXScrnSaver",
    "lsb"
  ],
  "depends": [
    "gawk",
    "git",
    "gconf2",
    "gconf-service",
    "gvfs-bin",
    "libc6",
    "libcap2",
    "libgtk2.0-0",
    "libudev0 | libudev1",
    "libgcrypt11 | libgcrypt20",
    "libnotify4",
    "libnss3",
    "libxtst6",
    "libxss1",
    "python",
    "xdg-utils",
    "gir1.2-gnomekeyring-1.0",
    "libgnome-keyring0"
  ]
}
```
**What did you expect to happen?**
We want to enable gconf2 on suse which has a different name.
**What actually happened?**
It couldn't compile (details below)
When I attempt to change requires from Gconf2 to Gconf2 | gconf2
```
rpmbuild -bb /tmp/electron-117628-9243-40sead.anleb/brave_0.20.0_x86_64/SPECS/brave.spec --target x86_64
error: line 9: Dependency tokens must begin with alpha-numeric, '_' or '/': Requires: GConf2 | gconf2, libXScrnSaver, lsb
```
This should work as of https://fedoraproject.org/wiki/Packaging:Guidelines#Rich.2FBoolean_dependencies

### PR DESCRIPTION
**Pull request by fcastilloec**
Fixes #74

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -280,6 +280,7 @@ Type: `Array[String]`
 Default: `["lsb", "libXScrnSaver"]`
 
 Packages that are required when the program starts, used in the [`Requires` field of the `spec` file](https://fedoraproject.org/wiki/How_to_create_an_RPM_package#Creating_a_SPEC_file).
+In order to use [boolean dependencies](http://rpm.org/user_doc/boolean_dependencies.html) you need `rpm >= 4.13`
```
Comment (malept) [inline README.md]: Period at end of sentence.

**Comment by fcastilloec** [comment]
Sorry, it was a little fast just so they can check the requirement for Travis.
Sure, let's put it there. I usually check any details on a module on the section that it's relevant to it.

**Comment by malept** [comment]
Sorry, I seem to have forgotten about this PR.

**Comment by fcastilloec** [comment]
No problem, I forgotten about it too

**Pull request merged**